packages/palette/src/elements/Skeleton/Skeleton.tsx

```diff
@@ -75,3 +76,31 @@ export const Skeleton: React.FC<SkeletonProps> = ({ children, ...rest }) => {
     </Box>
   )
 }
+
+export const SkeletonImage: React.FC<SkeletonProps & { blurhash?: string }> = ({
+  children,
+  blurhash,
+  ...rest
+}) => {
+  return (
+    <Box position="relative" {...rest}>
+      {!!blurhash && (
+        <BlurhashCanvas
```

Comment:
**question:** Can you confirm what the behavior is when `blurhash` is not supplied. Just falls back to the gray bg, yes?

Reply:
Yes! Here's a video for it from storybook

https://github.com/artsy/palette/assets/20655703/4e4d99b3-9b30-4182-a50d-0de2d5a26ac1